Index: src/GUI/Controller/NewSongViewController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/GUI/Controller/NewSongViewController.java b/src/GUI/Controller/NewSongViewController.java
--- a/src/GUI/Controller/NewSongViewController.java	
+++ b/src/GUI/Controller/NewSongViewController.java	
@@ -1,5 +1,8 @@
 package GUI.Controller;
 
+import BE.Song;
+import GUI.Model.PlaylistModel;
+import GUI.Model.SongModel;
 import javafx.event.ActionEvent;
 import javafx.fxml.FXML;
 import javafx.fxml.Initializable;
@@ -11,6 +14,9 @@
 
 public class NewSongViewController extends BaseController implements Initializable {
 
+    public ListView<Song> lstSong;
+    private SongModel songModel;
+
     @FXML
     private Button chooseFIle;
     @FXML
@@ -26,7 +32,7 @@
     @FXML
     private TextField txtFile;
     @FXML
-    private ComboBox<String> categoryDropdown;
+    private ComboBox categoryDropdown;
     
 
     @Override
@@ -34,19 +40,34 @@
         
     }
 
+    public void handleCategoryDropdown(ActionEvent actionEvent) {
+        MenuButton menuButton = new MenuButton("Category");
+        menuButton.getItems().addAll(new MenuItem("Pop"), new MenuItem("HipHop"));
+    }
+
     public void handleCancelSong(ActionEvent actionEvent) {
         Stage stage = (Stage) cancelSong.getScene().getWindow();
         stage.close();
     }
 
+
+
     public void handleSaveSong(ActionEvent actionEvent) {
+
+        String title = txtTitle.getText();
+        String artist = txtArtist.getText();
+        String category = categoryDropdown.getId();
+        int time = Integer.parseInt(txtTime.getText());
+
+        try {
+            songModel.createSong(title, artist, category, time);
+        } catch (Exception e){
+
+        }
     }
 
     @Override
     public void initialize(URL location, ResourceBundle resources) {
-        categoryDropdown.getItems().addAll("Pop", "HipHop", "Jazz", "Rap", "Rock", "Blues", "EDM",
-                "Hard Style", "Metal", "Alternative", "Classic", "Country", "R&B", "Soul");
-        categoryDropdown.getSelectionModel().selectFirst();
 
     }
 }
